### ISSUE TITLE
Getting a prop from a json saved in env

### ISSUE THREAD

**Issue by vikingprank** [rangav/thunder-client-support#1227]
Hello, 
Thanks for thunder, it is very nice.
The only thing I find bothersome is that, if I save the response of a request in json format in my env (with the tests tab), I can't easily access a prop of that json in another request.

For example, here is my response that I save in the env:

```json
{
    "city": "Paris",
    "country": "France",
    "formattedAddress": "66 Rue de la Grotte, 70000 Paris, France",
    "postalCode": "70000",
    "streetName": "Rue de la Grotte",
    "streetNumber": "66",
    "lat": "39.89835483",
    "long": "8.31536518"
}
```

Then in another request I'd like to use the lat & long props of the saved json:
In the query tab of the new request, I tried: 
`{{address | prop("lat")}}`
I also tried:
`{{address | map("long")}}`

But nothing is found, and the request is sent with empty values. From what I understood, it's because prop & map only work on arrays. My data is directly a json object.

I then saw that maybe I could use custom filters.
I dropped the demo script from here [https://github.com/rangav/thunder-client-support/blob/master/docs/custom-filters.js](url) in my workspace:
`backend\thunder-tests\thunder-custom-filters.js` 

and restarted vs code, but there doesn't seem to be a way to load custom scripts on query parameters. (BTW I can't find any custom scripts when I go to the tests tab and try to load a Post Request Script anyways, I'm probably missing something to configure this correctly.)

The current work around that I found is to save lat & long seperately in my env. But it is more work. Is there any other solution for me?

**Comment by rangav** [comment]
Hi @vikingprank 

The address is a saved as `string` in Env, so you cannot use the filter `prop("lat")`.

Will provide a new filter to parse to JSON first then you can use `prop` filter.

proposed solution e.g: - `{{address | parseJSON | prop("lat")}}`

`parseJSON` will be in the next update.


### Custom Scripts:

To use in query params - e.g: `{{address | customFilter}}`

sample filter code
```js
function customFilter(input) {
    let json = JSON.parse(input);
    let lat = json.lat;
    return lat;
}

module.exports = [customFilter]
```

**Comment by vikingprank** [comment]
Thank you for your response, the parseJSON solution sounds great, looking forward to the next update :)

I still don't understand how to get custom filters to be "detected".
In my vscode workspace settings, "save to workspace" is checked, and the "Workspace Relative Path" is set to "/backend" in my case, so I simply drop the custom filters script in "backend\thunder-tests". Is this the correct approach?

**Comment by rangav** [comment]
Please follow the steps here
https://github.com/rangav/thunder-client-support/blob/master/docs/filters.md#custom-filters

**Comment by vikingprank** [comment]
I just realized I missed the step of attaching the script to the collection! My bad !
thanks again

**Comment by rangav** [comment]
`parseJSON` filter is added and published to the marketplace, please update to v2.9.2

See all features released
https://github.com/rangav/thunder-client-support/releases/tag/v2.9.2

Please let me know your feedback.
